server/src/main/java/org/elasticsearch/tasks/Task.java

```diff
@@ -193,6 +193,29 @@ public Status getStatus() {
         return null;
     }
 
+    @Override
+    public String toString() {
+        return "Task{"
+            + "id="
+            + id
+            + ", type='"
+            + type
+            + '\''
+            + ", action='"
+            + action
+            + '\''
+            + ", description='"
+            + description
+            + '\''
+            + ", parentTask="
+            + parentTask
+            + ", startTime="
+            + startTime
+            + ", startTimeNanos="
+            + startTimeNanos
+            + '}';
```

Comment:
```suggestion
        return "Task{id=" + id + ", type='" + type + "', action='" + action + "', description='" + description + "', parentTask=" + parentTask + ", startTime=" + startTime + ", startTimeNanos=" + startTimeNanos + '}';
```
to avoid escaping. Needs to be reformatted